src/chi/components/alert/_toast.scss

```diff
@@ -3,35 +3,30 @@
 .chi {
   .m-alert {
     &.-toast {
+      align-items: start;
       max-width: 20rem;
       padding-bottom: 1.1875rem;
       padding-top: 0.75rem;
 
-      &::before {
-        font-size: $text-large;
-        margin-bottom: 0.4375rem;
-        padding: 0.25rem 0;
+      & > :first-child {
+        margin-top: 0.3125rem;
+        margin-left: 1rem;
+      }
+
+      & .a-h5 {
+        margin-top: 0;
+      }
+
+      & ~ .a-icon {
+        position: absolute;
```

Comment:
@davidojedacl try to avoid the position absolute here and wrap the icon and the text in a div. So you can use flex properties properly.